### ISSUE TITLE
Update lesson status

### ISSUE THREAD

**Issue by sstevens2** [datacarpentry/genomics-r-intro#81]
I'm wondering if this lesson should be updated to Beta status.  We've taught it a couple times now and I know there are others who have as well.

**Comment by ytakemon** [comment]
Thanks for bringing this up, @sstevens2 ! Beta seems appropriate according to the [lesson life cycle](https://cdh.carpentries.org/the-lesson-life-cycle.html). Let me bring this up to the Genomics R maintainer team when we meet next week and get back to you. Cheers

**Comment by ErinBecker** [comment]
Yes, I think beta would be appropriate. Please let me know what the team decides and I can help make sure everything gets updated with the new status.

**Comment by ewallace** [comment]
We're about to teach from this material in Edinburgh - anything our team can do to help on the alpha -> beta front?

**Comment by ewallace** [comment]
So @edcarp taught the material in Edinburgh and it went well. 
Lesson homepage: https://github.com/edcarp/2020-12-01-edinburgh-bio-online
Feedback here: https://pad.carpentries.org/2020-12-01-edinburgh-bio-online

I just put in pull request #95, which is intended to move lesson 5 up to beta, i.e. #44. 
Also @FlicAnderson taught episodes 1-4, and we aim to put in some more pull requests on those in the new year.

**Comment by sstevens2** [comment]
Note for when the maintainers are interested in making this beta, there needs to be an update the data Carpentry Lessons page too. Currently, this lesson is is listed in the "Lessons in Development" table and says it is Alpha there.

**Comment by zkamvar** [comment]
If I recall correctly, making this beta involves changing the tag from alpha to beta on the github landing page (note the right of the image):

![GitHub landing page for datacarpentry/genomics-r-intro](https://user-images.githubusercontent.com/3639446/103911622-1963a800-50bb-11eb-8035-f693659980ea.png)

**Comment by tobyhodges** [comment]
And the banner on the lesson pages can be updated by setting `life_cycle: "beta"` in `_config.yml`.

**Comment by naupaka** [comment]
To do for this change:

- [ ] Change tag on main repo page from alpha -> beta
- [ ] Change `life_cycle` yaml parameter on each episode page from alpha -> beta

**Comment by zkamvar** [comment]
> To do for this change:
> 
> * [ ]  Change `life_cycle` yaml parameter on each episode page from alpha -> beta

Note: this only needs to be changed in the `config.yaml` file, not each episode.

**Comment by naupaka** [comment]
@tobyhodges I think everything is updated on our end to have this represented as a beta not as alpha -- the only thing left is that the table [here](https://datacarpentry.org/lessons/#genomics-workshop) still shows as alpha. Not sure if that is auto-updated now that the tag has been changed or if it needs to be done manually?

**Comment by zkamvar** [comment]
IIRC, @ErinBecker needs to manually update that table

**Comment by tobyhodges** [comment]
Thanks for your patience as I get back on top of my GitHub notifications. I have just made the change.

**Comment by naupaka** [comment]
Awesome, thanks @tobyhodges!
